src/languageTools/PathConverters.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
This file is the same as `LanguageClient/Utils.js`, should they just use same file?

Reply:
LanguageClient here would work in the Node Context while Utils works in the CEF/Brackets context. The function APIs and logic are the same but there are subtle differences internally wherein the former one we use node.js built-ins and browser ones in the later.